release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -72,6 +72,32 @@ Previously, secret information was used as the OAuth access token and OAuth auth
 If you are upgrading your cluster to {product-title} 4.6, old tokens from {product-title} 4.5 will still have the secret information exposed in the object name. By default, the expiration for tokens is 24 hours, but this setting can be changed by administrators. Sensitive data can still be exposed until all old tokens have either expired or have been deleted by an administrator.
 ====
 
+[id="ocp-4-6-networking"]
+=== Networking
+
+[id="ocp-4-6-pod-network-connectivity-checks"]
+==== Pod network connectivity checks
+
+Operators can now configure `PodNetworkConnectivityCheck` resources to check each network connection from the Pods that are managed by the Operator. This allows you to more easily identify and troubleshoot issues with important network connections in your cluster.
+
+This resource keeps track of the latest reachable condition, the last 10 successes, the last 10 failures, and details about detected outages. The results are also logged and events are created when outages are detected and resolved.
+
+By default, the following network connections are checked:
+
+* Between the Kubernetes API server and:
+** the OpenShift API server service.
+** each OpenShift API server endpoint.
+** each etcd endpoint.
+** the internal API load balancer.
+** the external API load balancer.
+
+* Between the OpenShift API server and:
+** the Kubernetes API server service.
+** each Kubernetes API server endpoint.
+** each etcd endpoint.
+** the internal API load balancer.
+** the external API load balancer.
```

Comment:
I always forget the bullet list punctuation rules. Do you need periods at the end of each item in the list? IBM style guide seems to suggest not, in this case: _If list items only comprise sentence fragments, do not include any end puncutation._

Reply:
@mburke5678 I debated that and didn't have them at first. But I added them because of the "Between the OpenShift API server and" part, but I guess that's not truly a sentence either. I'll remove them.